src/libostree/ostree-repo.h

```diff
@@ -1218,6 +1218,7 @@ gboolean ostree_repo_prune_from_reachable (OstreeRepo             *self,
  * @OSTREE_REPO_PULL_FLAGS_UNTRUSTED: Do verify checksums of local (filesystem-accessible) repositories (defaults on for HTTP)
  * @OSTREE_REPO_PULL_FLAGS_BAREUSERONLY_FILES: Since 2017.7.  Reject writes of content objects with modes outside of 0775.
  * @OSTREE_REPO_PULL_FLAGS_TRUSTED_HTTP: Don't verify checksums of objects HTTP repositories (Since: 2017.12)
+ * @OSTREE_REPO_PULL_FLAGS_IMMUTABLE_FILES: Denotes that the files of local repository are immutable (Since: 2018.10)
```

Comment:
The Since here since dubious, given that we're already at 2018.12

Reply:
ah right. What was I thinking :/

Reply:
No, 2018.10 is right. 2018.9 was the last release.

Reply:
Oh, they aren't date-based? Ignore me